presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -44,6 +44,7 @@
     private int hashPartitionCount = 100;
     private String partitioningProviderCatalog = GlobalSystemConnector.NAME;
     private ExchangeMaterializationStrategy exchangeMaterializationStrategy = ExchangeMaterializationStrategy.NONE;
+    private boolean streamMarkDistinctWithMaterializedExchange;
```

Comment:
nit: What about `useStreamExchangeForMarkDistinct`?

Reply:
You are right technically that is what this does. I'll change to that.